internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/fastclient/FastClientIndividualFeatureConfigurationTest.java

```diff
@@ -125,11 +127,17 @@ public void testServerReadQuota() throws Exception {
       assertTrue(clientException.getMessage().contains("VeniceClientRateExceededException"));
     }
     boolean readQuotaRejected = false;
+    boolean errorRequest = false;
+
     for (MetricsRepository serverMetric: serverMetrics) {
       if (serverMetric.getMetric(readQuotaRejectedString).value() > 0) {
         readQuotaRejected = true;
       }
+      if (serverMetric.getMetric(errorRequestString).value() > 0) {
```

Comment:
Can we also add an unit test for `StatsHandler.write` if possible for the different `getResponseStatus`?

Reply:
Looks like to unit test we need to create dummy ctx which needs too many boilterplace class method overrides. Probably I will need to refactor the method to cleanly write unit test, which I am keeping outside the scope of this critical PR.